lib/src/dartdevc/workers.dart

```diff
@@ -8,16 +8,31 @@ import 'package:bazel_worker/driver.dart';
 import 'package:cli_util/cli_util.dart' as cli_util;
 import 'package:path/path.dart' as p;
 
+import '../log.dart';
+
 String get _scriptExtension => Platform.isWindows ? '.bat' : '';
 
+const _defaultMaxWorkers = 4;
+
+const _maxWorkersEnvVar = 'PUB_MAX_WORKERS_PER_TASK';
+
+final int _maxWorkersPerTask = int.parse(
+    Platform.environment[_maxWorkersEnvVar] ?? '$_defaultMaxWorkers',
+    onError: (value) => warning(
```

Comment:
I don't see where onError returns the default

Reply:
Lol good catch, it happened to work because the default for the bazel worker package is the same and it was falling back on that.

Reply:
done